src/server/api/forms/courses.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+
+const router = express.Router();
+const registrar = require('../../services/registrar');
+
+router.post('/', (req, res, err) => {
+  (async () => {
+    const { termCode } = req.body;
+    const { subjectAreaCode: department } = req.body;
+    try {
+      const result = await registrar.call((err) => ({
+        url: `sis/api/v1/Classes/${termCode}`,
+        params: {
+          offeredTermCode: termCode,
+          subjectAreaCode: department,
+          PageSize: 0,
+        },
+      }));
+      if (!result) {
+        throw new Error('No results');
```

Comment:
@tatikamsetti What will there user see if there's no results? I don't think this is being handled on the frontend.